docs/accessing-the-cluster.md

```diff
@@ -203,5 +301,40 @@ monitoring-influx-grafana-controller-0133o   10.244.3.4          influxdb
                                                                  grafana                 kubernetes/heapster_grafana:v0.4
 ```
 
+##<a name="somanyproxies"></a>So Many Proxies
+There are several different proxies you may encounter when using kubernetes:
+  1. The [kubectl proxy](#kubectlproxy)
+    - runs on a users desktop or in a pod
+    - proxies from a localhost address to the kubernetes apiserver.
+    - adds authentication headers
+    - does not provide load balancing in the future.
```

Comment:
What does this mean?

I don't think it's true. Each request to the proxy generates a new request to apiserver. There's nothing to stop this from being load balanced over multiple apiservers once we have HA.

It's also probably not important, so I suggest omitting.